e2e/app/workspace-page.ts

```diff
@@ -44,62 +52,62 @@ export default class Workspaces extends authenticatedpage {
     return await this.puppeteerPage.waitFor(selectors.dataSet, { visible: true });
   }
 
-  public element_question1_forProfitPurpose(): ProjectPurposeQuestion {
-    return new ProjectPurposeQuestion(this.puppeteerPage, 'For-Profit Purpose');
+  public question1ForProfit(): ComponentWebElement {
```

Comment:
bunch of methods rename to make shorter

Reply:
for grouping get-element functions. too many elements on page. naming convention probably will change in future.